README.md

```diff
@@ -363,6 +364,70 @@ We now need to tell CDT which toolchain to use when building the project. This w
 
 ![](docs/images/7_cmake_toolchain.png)
 
+<a name="SelectDifferentToolchain"></a>
+# Selecting Clang Toolchain
+In ESP-IDF Eclipse Plugin v2.7.0 you can build your project with Clang Toolchain.
```

Comment:
With ESP-IDF Eclipse Plugin v2.7.0 and higher you can build your project with Clang Toolchain

Reply:
fixed. Thanks